res/controllers/Electrix-Tweaker-scripts.js

```diff
@@ -908,15 +833,9 @@ ElectrixTweaker.modeButton = function (channel, control, value, status, group) {
 
 ElectrixTweaker.fader = function (channel, control, value, status, group) {
 	group = ElectrixTweaker.deck[group]
-	if (ElectrixTweaker.deckShift[group]) {
-		if (Math.abs(engine.getValue(group, 'rate') - (value - 64)/64) < .2) {
-			engine.setValue(group, 'rate', script.absoluteLin(value, -1, 1))
-		}
-	} else {
-		// soft takeover
-		if (Math.abs(value - script.absoluteNonLinInverse(engine.getValue(group, 'volume'), 0, .25, 1)) < 30) {
-			engine.setValue(group, 'volume', script.absoluteNonLin(value, 0, .25, 1))
-		}
+	// soft takeover
+	if (Math.abs(value - script.absoluteNonLinInverse(engine.getValue(group, 'volume'), 0, .25, 1)) < 30) {
```

Comment:
This was changed because having a shift function for a fader is confusing to use, which is the point of this PR. I can write a bit more descriptive comment though.